src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -344,9 +344,9 @@
     String getContentRaw();
 
     /**
-     * Gets the textual content of this message using {@link #getContentDisplay()} and then strips it of all markdown characters
-     * like {@literal *, **, __, ~~} that provide text formatting. Any characters that match these but are not being used
-     * for formatting are escaped to prevent possible formatting.
+     * Gets the textual content of this message using {@link #getContentDisplay()} and then strips it of all spoiler tags 
+     * and markdown characters like {@literal *, **, __, ~~, ||} that provide text formatting. Any characters that match these 
```

Comment:
The spoiler tags are also markdown characters so this is redundant

Reply:
Will revert